cosmosis/output/output_base.py

```diff
@@ -136,6 +136,13 @@ def parameters(self, *param_groups):
         if not self.begun_sampling:
             self._begun_sampling(params)
             self.begun_sampling=True
+
+        #Blind output by applying offsets
```

Comment:
There are a few other subclasses of output that will crash right now - the null output, cosmomc output, and fits output

Reply:
I modified these accordingly.